src/components/SatelliteManifestPanel/SatelliteManifestPanel.tsx

```diff
@@ -49,22 +49,32 @@ interface BooleanDictionary {
 const SatelliteManifestPanel: FunctionComponent<SatelliteManifestPanelProps> = ({
   data,
   isFetching,
-  user
+  userPermissions
 }) => {
-  const [columns] = useState([
-    { title: 'Name', transforms: [sortable], cellFormatters: [expandable] },
-    { title: 'Version', transforms: [sortable] },
-    {
-      title: (
-        <>
-          Simple Content Access
-          <SCAInfoIconWithPopover />
-        </>
-      ),
-      transforms: [sortable, cellWidth(20)]
-    },
-    { title: 'UUID', transforms: [sortable] }
-  ]);
+  const getColumns = (userPermissions: UserPermissions) => {
```

Comment:
Doesn't this function have direct access to the userPermissions prop?  Why did you add it as an arg to the method?

Reply:
yeah so it's just a habit from trying to write pure functions where possible. It's not actually a pure function in this case bc it still relies on components outside the function's scope. So I went ahead and changed it so the user object isn't passed in as a param.